resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "renkforce",
+        "platform_offset": [0, 0, 0],
+        "visible": true
+    },
+
+    "overrides": {
+        "adhesion_type": {
+            "default_value": "skirt"
+        },
+        "bottom_thickness": {
+            "default_value": 0.5
```

Comment:
Like I said [here](https://github.com/Ultimaker/Cura/issues/1316#issuecomment-274355866), it's wise to overwrite `"value"` instead of `"default_value"`. This setting change won't have any effect otherwise.